types/braintree-web-drop-in/index.d.ts

```diff
@@ -286,12 +286,11 @@ export function create(options: Options, callback: (error: object | null, dropin
 export function create(options: Options): Promise<Dropin>;
 
 // Global
-
 declare global {
-    const braintree: {
-        dropin: {
+    namespace braintree {
+        export const dropin: {
```

Comment:
There's currently a lint error because this `export` here is not needed. Can you fix that?

Reply:
Thanks for notify. I have fixed the error. Could you please take out some time to review @gabritto.